Example/Tests/GINIPDFDocumentTests.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  GINIPDFDocumentTests.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 9/14/17.
+//  Copyright © 2017 CocoaPods. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision
+
+class GINIPDFDocumentTests: XCTestCase {
+  
+    let pdfNonRotated = GINIPDFDocumentTests.loadPDFDocument(withName: "testPDF")
+    
+    func testNumberPages() {
+        let numberOfPages = pdfNonRotated.numberPages
+        
+        XCTAssert(numberOfPages == 1, "Number of pages should be 1 since the pdf only has one page")
+    }
+    
+    func testPDFNonRotated() {
+        let imageSize = pdfNonRotated.previewImage!.size
+        
+        XCTAssert(imageSize.height > imageSize.width, "Height must be greater than width since image has not rotationAngle and the pdf is in portrait orientation by default")
+    }
+    
+    func testPDFRotated90() {
+        let pdfDocument = GINIPDFDocumentTests.loadPDFDocument(withName: "testPDF-rotated90")
+        
+        let imageSize = pdfDocument.previewImage!.size
```

Comment:
One note about using "!" in tests: It's true that crashing is not a big deal (and even sometimes welcome) in unit tests. However, if an exception is thrown in the test body, Xcode will mark it as "an unexpected failure". You will still see which test failed, but the error message might not be as good (for instance, you wont see the beautiful assert message below). That being said, I still wouldn't consider using "!" here an error. But if it is easy to avoid it, you might reconsider.